packages/backend/src/preprocessor.py

```diff
@@ -102,23 +113,14 @@ def create_db_from_news_transcripts(news_json_directory):
 
         data = loader.load()
         text_splitter = RecursiveCharacterTextSplitter(
-            chunk_size=10000, chunk_overlap=5000
+            chunk_size=500, chunk_overlap=250
```

Comment:
@ayyubibrahimi does this look like a regression to you? @outlawhayden any reason for the change?